tools/xtask/src/qemu.rs

```diff
@@ -77,6 +75,19 @@ impl QemuCommand {
                         .arg("isa-debug-exit,iobase=0xf4,iosize=0x04");
                 }
 
+                let has_kvm = std::env::consts::ARCH == self.arch.to_string()
+                    && Path::new("/dev/kvm").exists();
+                if has_kvm {
+                    self.cmd.arg("-enable-kvm");
+                }
```

Comment:
This statement is similar to the one below, consider removing the other. Keep the comments.